bitcoin_client/ledger_bitcoin/exception/device_exception.py

```diff
@@ -6,8 +6,9 @@
 
 class DeviceException(Exception):  # pylint: disable=too-few-public-methods
     exc: Dict[int, Any] = {
+        0x5515: SecurityStatusNotSatisfiedError,  # returned by sdk in recent versions
```

Comment:
It seems a bit confusing to say recent and older without specific versions :sweat_smile:

Reply:
Yeah, but unfortunately it's not easy to track down the exact version; not critical, though - at least we fix the default current behavior.